Source/MDKUtilities/MDKUtilities.csproj

```diff
@@ -35,6 +35,7 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" />
```

Comment:
@malware-dev this isn't needed and can be reverted. I needed to add this in just so it'll compile.